import-export-cli/integration/testutils/policy_testUtils.go

```diff
@@ -75,6 +76,49 @@ func ValidateThrottlePolicyExportImport(t *testing.T, args *PolicyImportExportTe
 	validatePoliciesEquality(t, true, args, importedPolicy)
 }
 
+// Validates whether the throttling policy deletion is complete
+func ValidateThrottlingPolicyDelete(t *testing.T, args *PolicyImportExportTestArgs, username, password, policyType string) {
```

Comment:
Why do we need username and password as arguments here? I can't find the usages of those inside this function

Reply:
Fixed by [859ba91](https://github.com/wso2/product-apim-tooling/pull/928/commits/859ba919a58e2b9a690c34ebf23a137abba6f93b)